packages/realm-react/README.md

```diff
@@ -323,8 +323,149 @@ const SomeComponent = () => {
 }
 ```
 
+### Authentication Hooks
 
-#### Multiple Realms
+The following hooks can be used to authenticate users in your application.  They return authentication operations and a single result object which can be read to track the progress of the current result. More information about the specific auth methods can be found in the [Authenticate Users](https://www.mongodb.com/docs/realm/sdk/react-native/manage-users/authenticate-users) Documentation.
+
+## `useAuth`
+These hooks would typically be used in the `fallback` component of the `UserProvider`
+
+### `result`
+The result has the following structure:
+```tsx
+{
+  /**
+   * The current state of the operation.
+   * Enumerated by OperationState
+   */
+  state, // "not-started", "pending", "success", "error"
+
+  /**
+   * Convenience accessors, so users can write e.g. `loginResult.pending`
+   * instead of `loginResult.state === OperationState.Pending`
+   */
+  pending, // true or false
+  success, // true or false
+
+  /**
+   * The error returned from the operation, if any. This will only be populated
+   * if `state === OperationState.Error`, and will be cleared each time the
+   * operation is called.
+   */
+  error // Error based object or undefined
+}
+```
+
+This can be used to manage the state of the current login operation.
+
+### `logIn`
+Log in with a Realm.Credentials instance. This allows login with any authentication mechanism supported by Realm. If this is called when a user is currently logged in, it will switch the user.  Typically the other methods from `useAuth` would be used.
```

Comment:
```suggestion
Log in with a Realm.Credentials instance. This allows login with any authentication provider supported by Atlas App Services. If this is called when a user is currently logged in, it will switch the user.  Typically the other methods from `useAuth` would be used.
```

Reply:
> Typically the other methods from `useAuth` would be used.

I don't think I understand this.

Reply:
Neither do I 😅